docs/3d_rendering.md

```diff
@@ -0,0 +1,123 @@
+## 3D Rendering
```

Comment:
**chore:** Make sure to add a link to this to `docs/index.md`, similar to the `bugs_and_glitches` doc. I would create a new subsection in the `index` to house documents like this, e.g. `Subsystems`